waku/v2/protocol/filter/client.go

```diff
@@ -159,16 +160,23 @@ func (wf *WakuFilterLightNode) onRequest(ctx context.Context) func(s network.Str
 			wf.metrics.RecordError(decodeRPCFailure)
 			return
 		}
-
-		if !wf.subscriptions.Has(s.Conn().RemotePeer(), messagePush.PubsubTopic, messagePush.WakuMessage.ContentTopic) {
-			logger.Warn("received messagepush with invalid subscription parameters", logging.HostID("peerID", s.Conn().RemotePeer()), zap.String("topic", messagePush.PubsubTopic), zap.String("contentTopic", messagePush.WakuMessage.ContentTopic))
+		//For now returning failure, this will get addressed with autosharding changes for filter.
+		if messagePush.PubsubTopic == nil {
+			logger.Error("empty pubSub Topic", zap.Error(err))
```

Comment:
No err is generated here
```suggestion
			logger.Error("empty pubsub topic")
```